test/fixtures.yml

```diff
@@ -439,9 +439,9 @@ garanti:
 
 global_collect:
   merchant_id: 2196
-  api_key_id: c91d6752cbbf9cf1
-  secret_api_key: xHjQr5gL9Wcihkqoj4w/UQugdSCNXM2oUQHG5C82jy4=
-
+  api_key_id: b2311c2c832dd238
+  secret_api_key: Av5wKihoVlLN8SnGm6669hBHyG4Y4aS4KwaZUCvEIbY=
```

Comment:
Is this supposed to be for a new fixtures entry for `global_collect_direct`?

Reply:
Not really, these are the current credentials for `GlobalCollect`. For `Ogone` Direct API the credentials are in 1password, but I think that is not a good idea to include it into this commit